front/src/modules/ui/theme/constants/colors.ts

```diff
@@ -22,6 +22,8 @@ export const grayScale = {
   gray0: '#ffffff',
 };
 
+export type ThemeColor = keyof typeof color;
```

Comment:
actually, the only ThemeColor that we should be able to use are the "primary" ones:  yellow | green... and not yellowXXX